modules/ocf_kubernetes/manifests/master.pp

```diff
@@ -20,21 +20,66 @@
   $etcd_archive = "etcd-v${etcd_version}-linux-amd64.tar.gz"
   $etcd_source  = "https://github.com/etcd-io/etcd/releases/download/v${etcd_version}/${etcd_archive}"
 
+  class { 'ocf_kubernetes::package::first_stage':
+    stage => first,
+  }
```

Comment:
Why was this added? What is it needed for?

Reply:
Hmm, this might be an artifact of earlier changes. I'll see if it's safe to delete this.